src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
+$main-color: #060b35;
+$main-secondary: #616070;
+$buttom-color: #00acdc;
+
 body {
-  margin: 0;
+  text-align: center;
+  margin: 10px 100px;
+}
+
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    margin: 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+    text-align: left;
+  }
+
+  &__code {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $main-secondary;
+    text-align: left;
+  }
+
+  &__rating {
+    margin: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-price {
+      margin: 0;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    text-align: center;
+
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $main-secondary;
+    }
+
+    &-buy {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: $buttom-color;
+    border: 1px solid;
+    border-color: #00acdc;
+    border-radius: 5px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    &:hover {
+      border-color: $buttom-color;
+      background-color: #fff;
+      color: $buttom-color;
+    }
+  }
+
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  &:not(:first-child) {
+    margin-left: 4px;
+  }
+
+  &--4 :nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
 }
```

Comment:
use the entire stars styling from the previous task, to have a possibility of changing active stars' quantity